src/scss/_primary-nav.scss

```diff
@@ -3,6 +3,8 @@
 /// @param {String} $draw-breakpoint At what breakpoint to show the draw hamburger.
 /// @param {Boolean} $drop-down - Whether to include styles to support the dropdown nav.
 @mixin _oHeaderServicesPrimaryNav($drawer-breakpoint, $drop-down) {
+
+	@include _oHeaderServicesDrawer($drawer-breakpoint, $drop-down);
```

Comment:
need to manually verify this didn't cause a visual regression

Reply:
manual testing appears ok